package.json

```diff
@@ -7,8 +7,11 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "bootstrap": "^4.5.3",
+    "firebase": "^8.2.2",
+    "firebase-admin": "^9.4.2",
```

Comment:
Hmm... not sure if this should be in a client app. The admin sdk is usually for building stuff that modifies your firebase infrastructure directly. Not safe to put in a client app.